packages/client/test/integration/StreamrClient.test.ts

```diff
@@ -54,30 +61,10 @@ describeRepeats('StreamrClient', () => {
         expect(errors).toHaveLength(expectErrors)
     })
 
-    let stream: Stream
-    let publishTestMessages: ReturnType<typeof getPublishTestMessages>
-
-    // These tests will take time, especially on Travis
-    const TIMEOUT = 30 * 1000
-    const WAIT_TIME = 600
-
-    const createStream = async ({ requireSignedData = true, ...opts }: any = {}) => {
-        const s = await client.createStream({
-            id: createRelativeTestStreamId(module),
-            requireSignedData,
-            ...opts,
-        })
-        await until(async () => { return client.streamExistsOnTheGraph(s.id) }, 100000, 1000)
-        expect(s.id).toBeTruthy()
-        expect(s.requireSignedData).toBe(requireSignedData)
-        return s
-    }
-
     beforeEach(async () => {
         client = await createClient()
```

Comment:
Do we actually need client with gas here?

Reply:
Fixed